resolwe/process/resources.py

```diff
@@ -0,0 +1,16 @@
+"""Resource consumption estimators for processes."""
+
+from collections import defaultdict
+
+PROCESS_RESOURCES = defaultdict(dict)
+
+
+def estimator(process_slug, resource):
```

Comment:
In the rest of the code the `wrapt.decorator` is usually used to avoid the inner method.

Reply:
does not apply here